cmd/termination-handler/main.go

```diff
@@ -37,7 +36,7 @@ func main() {
 	klog.InitFlags(nil)
 	logger := klogr.New()
 
-	pollIntervalSeconds := flag.Int64("poll-interval-seconds", 5, "interval in seconds at which termination notice endpoint should be checked (Default: 5)")
+	_ = flag.Int64("poll-interval-seconds", 5, "interval in seconds at which termination notice endpoint should be checked (Default: 5)")
```

Comment:
does this flag get registered as a side effect of calling `flag.Int64` ?

i'm just curious if we don't need to save it, can we just drop it?

Reply:
We could, but this is the part which is manually set in `sync.go` by the MAO as one of the mandatory arguments. That will require a patch on its own to rely on default values. It would be preferable to find alternatives to that PR in other providers.